### PR TITLE
Add Travis CI and Coveralls

### PR DESCRIPTION
**Pull request by matheusps**
#### What is the purpose of this pull request?
As the title says.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,70 +172,78 @@ render() {
...
-| `perPage` | `number/object` | :no_entry_sign: | `1` | Amount of slides to be on the screen, if a number is passed, then that's the number of slides that will be shown, if an object with breakpoints is passed, then the component will check the size of the screen to see how many slides will be on the screen at the same time |
-| `resizeDebounce` | `number` | :no_entry_sign: | `250` | Resize debounce timer in milliseconds |
-| `rootTag` | `string` | :no_entry_sign: | `'div'` | Tag to be rendered in the root of the slider |
-| `sliderFrameTag` | `string` | :no_entry_sign: | `'ul'` | Tag to be rendered in the slider frame element |
-| `threshold` | `number` | :no_entry_sign: | `50` | Minimum of pixels to drag until the slider change the `currentSlide` |
+
+| Prop name                  | Type            | isRequired         | defaultValue                            | Description                                                                                                                                                                                                                                                                   |
```
Comment (lucasecdb) [inline README.md]: why this change?

Comment (matheusps) [reply README.md]: Oops... Auto format on saving, I'll take it back

**Review by lucasecdb — APPROVED** [review]

**Review by GustavoMA — APPROVED** [review]

**Review by theomoura — APPROVED** [review]
